### DISCUSSION TITLE
Explort "Select TG" Preferences

### DISCUSSION THREAD

**Discussion by tadscottsmith** [chuot/rdio-scanner#311] [General]
Is there a way that we could export our current client side talk group listening preferences that are selected from the "SELECT TG" button? My browser crashed when I had "HOLD TG" enabled, and somehow it seems my local application storage in my browser lost all of my preferences. It would be helpful to be able to back these up and re-import them occasionally. Would also be nice to move listening preferences between computers in lieu of a "profile" type setting saved on the server side.

**Comment by robertlynch3** [comment]
[In 6.6](https://github.com/chuot/rdio-scanner/blob/master/CHANGELOG.md#version-66) you can do `http://host:port/?id=<string>` which will save the selected talkgroups. I believe it works between browsers.

**Comment by chuot** [comment]
Persisting the talkgroups selection into the browser local storage when holding on a system or a talkgroup shouldn't be a problem. I'll add this feature to v7. As for the profiles, Robert is right, adding ?id=<string> to the URL kinda creates separated profiles in regards to the talkgroups selection, which are also stored into the browser local storage. However, it doesn't work through different browser, just between tabs.
